x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix.tsx

```diff
@@ -442,6 +522,28 @@ export const ScatterplotMatrix: FC<ScatterplotMatrixProps> = ({
                 </EuiFormRow>
               </EuiFlexItem>
             )}
+            {splom ? (
+              <EuiFlexItem grow={false}>
+                <EuiLink
+                  onClick={async () => {
+                    const customVisLink = getCustomVisualizationLink();
+                    await application.navigateToApp('visualize#', {
+                      path: customVisLink.path,
+                      openInNewTab: true,
+                    });
+                  }}
+                  data-test-subj="mlSplomoExploreInCustomVisualizationLink"
+                >
+                  <EuiIconTip
+                    content={i18n.translate('xpack.ml.splom.exploreInCustomVisualizationLabel', {
+                      defaultMessage: 'Explore scatterplot charts in custom visualization',
```

Comment:
Can we use the term 'Vega' in here, such as `Explore scatterplot charts in Vega based custom visualization`?

Reply:
Updated copy in e14f32ff3f779436910ec8787c521251916a7972